pkg/accountmanager/iam.go

```diff
@@ -0,0 +1,184 @@
+package accountmanager
+
```

Comment:
this is adapted from existing code, no?

Reply:
Yes, basically reworking what is in the policy and role manager